firefly_client/firefly_client.py

```diff
@@ -120,7 +120,6 @@ def __init__(self, host=_my_localhost, channel=None, basedir='firefly', html_fil
         self.listeners = {}
         self.channel = channel
         self.session = requests.Session()
-        self.headers = {'FF-channel': self.channel}
```

Comment:
Is `self.headers` not longer used?

Reply:
`self.headers` is used but the contents of `FF-channel` and `FF-connID` are set inside the `received_message` method.  The deleted line was added be @tgoldina and it works when `channel` is explicitly set, but not with the default `channel=None`.

Reply:
Well, I did not added this line. I just changed the header name from 'FF_FF-channel' to 'FF-channel', because Firefly knows nothing about the first. But, you are right, it's probably OK to remove this line, because we do set headers when web socket connection is established.